src/parser/demonhunter/havoc/modules/Abilities.js

```diff
@@ -102,7 +86,7 @@ class Abilities extends CoreAbilities {
         castEfficiency: {
           suggestion: true,
           recommendedEfficiency: 0.85,
-          extraSuggestion: <>Pool your fury before hand to maximizing casting <SpellLink id={SPELLS.CHAOS_STRIKE.id} icon /> / <SpellLink id={SPELLS.ANNIHILATION.id} icon /> during its buff window.</>,
+          extraSuggestion: <>Pool your Fury before hand to maximizing casting <SpellLink id={SPELLS.CHAOS_STRIKE.id} icon /> / <SpellLink id={SPELLS.ANNIHILATION.id} icon /> during its buff window.</>,
```

Comment:
Icon property doesn't need to be here (this might actually be in more places in this file, worth checking out?)
```suggestion
          extraSuggestion: <>Pool your Fury before hand to maximizing casting <SpellLink id={SPELLS.CHAOS_STRIKE.id} /> / <SpellLink id={SPELLS.ANNIHILATION.id} /> during its buff window.</>,
```

Reply:
removed